doc/api/vm.md

```diff
@@ -152,30 +188,28 @@ changes:
 * Returns: {any} the result of the very last statement executed in the script.
 
 Runs the compiled code contained by the `vm.Script` object within the given
-`contextifiedObject` and returns the result. Running code does not have access
-to local scope.
+`context` and returns the result. Running code does not have access to local
+scope.
 
 The following example compiles code that increments a global variable, sets
 the value of another global variable, then execute the code multiple times.
-The globals are contained in the `context` object.
+The globals are contained in the `context`'s global object.
 
 ```js
 const util = require('util');
 const vm = require('vm');
 
-const context = {
-  animal: 'cat',
-  count: 2
-};
+const context = new vm.Context();
+context.global.animal = 'cat';
+context.global.count = 2;
```

Comment:
they could just do Object.assign themselves.

Reply:
Well yes, but that's much clunkier than allowing to pass some initial values to global as that's what will usually be the case anyway considering that having different `global` is one of the features/use-cases of Context. Having to write `Object.assign` after you have just created an object to change the values/options on that object is kind of annoying in my opinion and delivers the intent of the developer much worse.
```js
const ctx = new vm.Context();
Object.assign(ctx.global, { a: 42, b: 13 });
```
instead of just
```js
const ctx = new vm.Context({
  global: { a: 42, b: 13 },
});
```

Reply:
If I added that it would just be an Object.assign anyway. I think it represents the intent of the user better to make them assign the properties how they expect them to be assigned.